app/views/posts/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>New post!</h1>
+<%= form_with model: [@user, @post], url: user_posts_path(@user), method: :post, local: false, scope: :post do |f| %>
+  <div>
+    <%= f.label :title, 'Post Title' %>
+    <%= f.text_field :title %>
+  </div>
+  <div>
+    <%= f.label :text, 'Post Text' %>
+    <%= f.text_area :text %>
+  </div>
+  <%= f.submit 'Create Post' %>
+<% end %>
```

Comment:
- [ ] @alexansaa, great effort working on your problem and you have done wonderfully well implementing most of the things required in the project. However, there are still some things affecting the approval of the project.

1.  In the post routes, I could not see the form I can use to add a `post` to the user, Also I could not access the `comment` to add a comment to the post as well.
Below is a video demonstration of what the website looks like. 👇 

[screen-capture (5).webm](https://github.com/alexansaa/my_blog_app/assets/78634841/f1d2548a-4987-4901-929c-103b150ac892)

Reply:
Hi, I have to say that there was no error like what you have commented. I have checked my project twice and I have found other errors, but not what you have posted. Please, kindly check if the branch in your local copy is correctly set to "current_user_and_forms" because what you are showing seems to be the "dev" branch which has the implementations from the last milestone where there were any forms. Thank you!

Reply:
@alexansaa, If you check the video I sent you, you can notice that I was on the `current_user_and_forms` branch where I was checking all your work...And as you said, it might be the error you worked on that affected the route to display the form...
But that's fine, I am happy you were able to solve the issue.
Also, I remember I reached out to you concerning the behavior of the project, and you acknowledged that the problem might be from your end.